versions.json

```diff
@@ -135,6 +135,27 @@
         ],
         "fpm_directories": ["bin","doc","share"]
     },
+    "cpr": {
+        "commitish": "1.3.0",
+        "version_string": "1.3.0",
+        "license": "MIT",
+        "consortium_build_number": "0",
+        "externals_root": "opt/irods-externals",
+        "build_steps": [
+            "git submodule update --init",
+            "rm -f CMakeCache.txt;TEMPLATE_CMAKE_EXECUTABLE -G 'Unix Makefiles' -DBUILD_SHARED_LIBS=True -DCMAKE_CXX_COMPILER=TEMPLATE_CLANGPP_EXECUTABLE -DCMAKE_C_COMPILER=TEMPLATE_CLANG_EXECUTABLE -DCMAKE_INSTALL_RPATH=/TEMPLATE_BOOST_RPATH\\;/TEMPLATE_CLANG_RUNTIME_RPATH -DCMAKE_INSTALL_PREFIX=TEMPLATE_INSTALL_PREFIX -DBOOST_ROOT=TEMPLATE_BOOST_ROOT -DCMAKE_CXX_FLAGS='-std=c++14 -nostdinc++ -ITEMPLATE_CLANG_CPP_HEADERS' -DCMAKE_EXE_LINKER_FLAGS='-stdlib=libc++ -LTEMPLATE_CLANG_CPP_LIBRARIES -lc++abi' -DCMAKE_SHARED_LINKER_FLAGS='-stdlib=libc++ -LTEMPLATE_CLANG_CPP_LIBRARIES -lc++abi' -DCMAKE_MODULE_LINKER_FLAGS='-stdlib=libc++ -LTEMPLATE_CLANG_CPP_LIBRARIES -lc++abi' .",
+            "make -jTEMPLATE_JOBS"
+            ],
+        "external_build_steps": [
+            "ls -l ",
+            "mkdir -p TEMPLATE_INSTALL_PREFIX",
+            "cp -rf ./lib TEMPLATE_INSTALL_PREFIX",
+            "cp -rf ./bin TEMPLATE_INSTALL_PREFIX",
+            "cp -rf ./include TEMPLATE_INSTALL_PREFIX",
```

Comment:
is this right?    putting all these files into the same place?

Reply:
The changes are from November; I'll look through them again, but am fairly sure it was because cpr was not setting up its directories where we could find them for later builds.

Reply:
In case there was confusion the directories themselves get copied to the same place; their contents don't get intermingled (see cp -r)